### PR TITLE
[DO NOT MERGE] Org avatars

### PR DESCRIPTION
**Pull request by ryanpbrewster**
@leogrim @andrewconner @Colin-Lane @yingjieMiao 

This compiles and passes tests, but if I'm honest I don't really understand the code I've written.
Also I completely ignored analytics.

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/OrganizationAvatarCommander.scala [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+  def getUrl(orgAvatar: OrganizationAvatar): String
+  def getBestImageForOrganization(orgId: Id[Organization], idealSize: ImageSize): Option[OrganizationAvatar]
+  def getBestImageForOrganizations(orgIds: Set[Id[Organization]], idealSize: ImageSize): Map[Id[Organization], OrganizationAvatar]
+  def uploadOrganizationAvatarFromFile(image: TemporaryFile, orgId: Id[Organization], position: ImagePosition, source: ImageSource, userId: Id[User], requestId: Option[Id[OrganizationAvatarRequest]] = None)(implicit content: HeimdalContext): Future[ImageProcessDone]
+  def positionOrganizationAvatar(orgId: Id[Organization], position: ImagePosition): Seq[OrganizationAvatar]
+  def removeImageForOrganization(orgId: Id[Organization], userId: Id[User])(implicit context: HeimdalContext): Boolean // Returns true if images were removed, false otherwise
+}
+
+@Singleton
+class OrganizationAvatarCommanderImpl @Inject() (
```
Comment (yingjieMiao) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/OrganizationAvatarCommander.scala]: add comment: Copied wholesale from LibraryImageCommander. 

Wondering if we can reuse code (or DRY the code), by factoring out some abstract class or trait. (Maybe not this PR?)

Keep this much copied code synced at two place can become a nightmare. 

It's really hard to tell what's new, what's modified, etc.

Looking at the difference, seems that there are repos of `org` or `library`. I guess they are just used to figure out some image meta data (id, path, size, etc). At least retrieving from `imageStore` could be factored out?

Not really familiar with the codes, throwing some ideas.

Comment (yingjieMiao) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/OrganizationAvatarCommander.scala]: As a reference: `FixedSetPersonaRecoPool` , `URIFixedSetPersonaRecoPool`, `LibraryFixedSetPersonaRecoPool`

Not necessarily a clean or good example. Something for inspiration.

Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/OrganizationAvatarCommander.scala]: I definitely think that there is _something_ here that can be factored out. I chatted with @leogrim about it and I have some ideas.

Comment (leogrim) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/OrganizationAvatarCommander.scala]: Yep, I would probably close this PR and start from scratch looking at the code in `Rover` instead.

**Pull request closed without merging**